.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 coverage/
-.twilio-functions
\ No newline at end of file
+.twilio-functions
+.env
```

Comment:
We actually need the `.env` files right now since they inform the `twilio-run` tool (aka what is being called when you run `twilio serverless:...` what env variables should be placed as placeholders into the `.env` file. https://github.com/twilio-labs/twilio-run/blob/master/src/templating/filesystem.ts#L144

Reply:
I can see the benefit of this though. So maybe the right solution here should be first to make `twilio-run` compatible with `.env.example` files before changing them here.

Reply:
@dkundel The instructions include creating your `.env` file locally, meaning the it will exist when `twilio serverless:...` is called: https://github.com/thorsten-stripe/function-templates/tree/stripe-sms-receipt/stripe-sms-receipt#environment-variables

Reply:
Oh I see it needs to be there when running `twilio serverless:init`, gotcha, will revert.